helm/chart_loader.go

```diff
@@ -86,8 +102,8 @@ func (cl *remoteChartLoader) fetchArchive(ctx context.Context, archiveURL string
 	return archiveResp, nil
 }
 
-func defaultBackoff(ctx context.Context) backoff.BackOffContext {
-	return backoff.WithContext(backoff.WithMaxRetries(backoff.NewConstantBackOff(1*time.Second), 5), ctx)
+func defaultBackoff() wait.Backoff {
+	return waitext.NewConstantBackoff(1 * time.Second)
```

Comment:
I feel like this is not very useful function as well. Could you consider using `waitext` directly here as well?